int8 type
---
bool success
string status